tools-v2/internal/utils/row.go

```diff
@@ -109,6 +109,8 @@ const (
 	ROW_TOTAL           = "total"
 	ROW_TYPE            = "type"
 	ROW_USED            = "used"
+	ROW_UNHEALTHY_COPYSET_RATIO = "unhealthy coyset ratio"
+	ROW_UNHEALTHY_COPYSET_COUNT = "unhealthy copyset count"
```

Comment:
format
Alignment, and I think, only keep `ROW_UNHEALTHY_COPYSET`, the following output is also good:
|23(10%)|